packages/pieces/localai/src/index.ts

```diff
@@ -0,0 +1,32 @@
+import { PieceAuth,Property, createPiece } from '@activepieces/pieces-framework';
+import { askLocalAI } from './lib/actions/send-prompt';
+
+export const localaiAuth = PieceAuth.CustomAuth({
+  
+  description: `
```

Comment:
Let's remove the description for the auth

Reply:
description removed